### ISSUE TITLE
Keyboard Layout Editor image for Miryoku-QWERTY (and other layouts?)

### ISSUE THREAD

**Issue by lightnin** [tompi/cheapino#95]
Hello good people -

I've decided to use my wonderful cheapino to try and learn Miryoku Qwerty. I have flashed the Qwert Miryoku firmware provided on the release page. However, I'm having difficulty figuring out a way to make a printable image of the keymap as a reference to help me learn (to be placed below my monitor).

www.keyboard-layout-editor.com seems like the right tool to generate such an image, as it allows for representations of multiple layers on the same keys, which is nice for making a compact reference image similar to this one, of plain colemak? miryoku.

![keyboard-layout-mir](https://github.com/user-attachments/assets/b9e34b45-0387-4cda-b78b-be9acc0614e2)

However, I can see that the Cheapino Qwerty Miryoku firmware doesn't match this Miryoku in some ways. Can anyone recommend how to go about creating an image on www.keyboard-layout-editor.com based on the Miryoku Qwerty firmware provided for the Cheapino? 

If it's simple enough I'll be happy to do it for this layout and any other ones released, as I think it will be a useful reference for anyone trying to learn a new keymap.

Tak!

p.s. Additional reference: https://www.reddit.com/r/ErgoMechKeyboards/comments/182nwwc/readable_miryoku_reference/

**Comment by lightnin** [comment]
I just modified the layout above to make it QWERTY, easily enough. However, the Miryoku Qwerty available on the release page doesn't seem to match the layer behavior shown in the image. Perhaps there are differences because I'm on a Mac. But the media, num, and nav layers behave mostly as espected, just not copy paste. But the mouse and fun layers don't seem to behave quite right. 
![keyboard-layout-qwertyMiryoku](https://github.com/user-attachments/assets/51c82c13-fcc3-436c-8a76-bd0e47e57ee9)

**Comment by tompi** [comment]
Hey, thats strange, I built the miryoku from the github repo...
Do you have an example of a key/sequence that is not as pictured?

**Comment by lightnin** [comment]
Well, I think it's pilot error. I was confused about the symbols. It looks like everything is right save the (mouse or sim?) copy paste symbols on U I and O. Would I need to recompile with MIRYOKU_CLIPBOARD=MAC to fix that?

**Comment by tompi** [comment]
Ah, Okay. Im not sure how that works, but since ctrl c/v is cmd c/v on Mac, it makes sense there might be some diff there...

Try recompiling. Let me know if you struggle setting it up, i remember there was an issue about this a while ago...

**Comment by lightnin** [comment]
Yep, I went through quite an odyssey, but did manage to get things working with qmk. However, I think it makes the most sense to just use your stock firmware even if the copy / paste keys don't work, and try to stay close to Miryoku in hopes that it's an emerging standard. So the graphic I shared a few posts back should do the trick for me (and feel free to post it somewhere as a resource for others, if you think it would be helpful to people learning.) Thanks!
